fr.inria.Coccinelle.metainfo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2024 INRIA, University of Copenhagen DIKU -->
+<component type="console-application">
```

Comment:
console applications are a poor fit with Flatpak.

Reply:
Yeah, I am aware of https://github.com/flatpak/flatpak/issues/1188#issuecomment-1073145867. Unfortunately, Flatpak is the only option if we want wide support.